apps/src/StudioApp.js

```diff
@@ -955,6 +955,17 @@ StudioApp.prototype.toggleRunReset = function(button) {
     reset.style.display = !showRun ? 'inline-block' : 'none';
     reset.disabled = showRun;
   }
+  if (this.isUsingBlockly() && !this.config.readonlyWorkspace) {
+    if (showRun && this.config.app !== 'craft') {
```

Comment:
Mind adding a quick comment? The if/else statements seem a little confusing.

Reply:
Also, flipping the logic so you use `this.config.app === 'craft'` rather than `!==` may help clear things up.